src/quo2/components/avatars/account_avatar/component_spec.cljs

```diff
@@ -83,7 +83,7 @@
         :width           (:size opts)
         :borderRadius    (style/get-border-radius (:size opts))
         :borderWidth     0.8
-        :backgroundColor (colors/custom-color-by-theme (:customization-color opts) 50 50 10 10)})
+        :backgroundColor (colors/resolve-color (:customization-color opts) :light 10)})
```

Comment:
I like the word `resolve` you chose, and I think the code would be just as clear with the more succinct `(colors/resolve ...)`. If you do this, you'll need to use `(:refer-clojure :exclude [resolve])`.